react-events-and-state/src/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom/client';
+
+export default function CustomButton(props) {
```

Comment:
No need to export it since it's not in a separate file.

I suggest you use destructuring syntax for the props, but not required.